src/utils.ts

```diff
@@ -283,13 +283,4 @@ const findPluginByParser = (parserName: string, options: ParserOptions) => {
     : tsPlugin.parsers?.[parserName];
 };
 
-export {
-  convertToModernType,
```

Comment:
I think exporting like previous code is so readable, you could find what is exported without scrolling in whole file

Reply:
I prefer this method since you can tell if a function is exported or not by just looking at the function. You don't have to scroll all the way to the bottom of the file to figure it out. It also cuts down a bit on duplication of the function name.

If you prefer the other method though then I can revert this change. It is your project after all.

Reply:
change reverted